test/templates/config.ru

```diff
@@ -0,0 +1,63 @@
+require_relative "fake_middleware"
+
+$current_dir = File.dirname(__FILE__)
```

Comment:
No need for this to be a global. You can avoid the scope gates and use it as a top-level local. Also, look up the [`Pathname`](http://ruby-doc.org/stdlib-2.2.2/libdoc/pathname/rdoc/Pathname.html) class in the standard lib.